packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -66,6 +66,7 @@ export class OdspDocumentService implements IDocumentService {
         socketIoClientFactory: () => Promise<SocketIOClientStatic>,
         cache: IOdspCache,
         hostPolicy: HostStoragePolicy,
+        getSharingLinkToken?: SharingLinkTokenFetcher,
```

Comment:
The signatures of getStorageToken/getWebsocket token are in a callback format that takes in TokenFetchOptions and returns the promise of the string. Is there a reason to depart from that API shape? I can imagine it'd be confusing for hosts to have to implement token fetching in different ways

Reply:
This is a callback - "SharingLinkTokenFetcher"

Reply:
Ah I see it! It does diverge from the paradigms a little in that it can return more than just a string and options isn't the first param, maybe we can consolidate these? Also maybe we should also define the other two callbacks in a similar way so it doesn't involve searching through a bunch of different files to figure out how to pass in tokens to this method, haha.

Reply:
Made it consistent with others. Return type is also same now. I realized I want to make it consistent with other 2 and not the other way around.